docs/source/DeveloperGuide/APIs/index.rst

```diff
@@ -0,0 +1,111 @@
+
+Consider the structure:
+-----------------------
+
+The main change introduced here is the separation in a :py:class:`~ DIRAC.Core.Tornado.Server.private.BaseRequestHandler` class of the basic logic for authorizing the request, search and run the target method in the thread.
+Thus TornadoService and TornadoREST following BaseRequestHandler can describe RPC and REST interfaces respectively::
+
+    BaseRequestHandler
+    |
+    |\_TornadoService
+    |
+     \_TornadoREST
+
+From the scheme it is intuitively clear that on the basis of BaseRequestHandler it is possible to create other framework if flexibility of existing is not enough.
+
+New components:
++++++++++++++++
+
+  - :py:class:`~ DIRAC.Core.Tornado.Server.private.BaseRequestHandler` contains the basic logic described here :ref:`httpsTornado` except for the access interface.
+  - :py:class:`~ DIRAC.Core.Tornado.Server.TornadoREST` framework for writing REST API for DIRAC systems
+
+What is the purpose of this?
+----------------------------
+
+The purpose of this component is to build DIRAC system status access interfaces based on HTTP requests outside the fixed RPC request interface described in TornadoServices.
+
+The main reason for the implementation of this feature is the need to implementation of API for DIRAC Authorization Server based on OAuth2 framework,
+which in turn is an integral part of the implementation of OAuth2 DIRAC authorization.
+So currently, there is only one API implementation for the Framework system, which is :py:class:`~ DIRAC.FrameworkSystem.API.AuthHandler`.
+For the same reason, the ability to authorize with an access token (see https://datatracker.ietf.org/doc/html/rfc6750#section-2) was added to the authorization steps.
+
+How to write APIs
+-----------------
+
+The following is an example of a simple API:
+
+.. literalinclude::
+
+    from DIRAC.Core.Tornado.Server.TornadoREST import TornadoREST
+    from DIRAC.ConfigurationSystem.Client.Helpers import Registry
+    class yourEndpointHandler(TornadoREST):
+
+      SYSTEM = "Configuration"
+      LOCATION = "/registry"
+
+      @classmethod
+      def initializeHandler(cls, infosDict):
+        ''' Called only once when the first request for this handler arrives
+        '''
+        pass
+
+      def initializeRequest(self):
+        ''' Called at the beginning of each request
+        '''
+        pass
+
+      # Specify the default permission for the method
+      auth_users = ['authenticated']
+
+      def web_users(self, count: int = 0):
+        ''' Your method. It will be available for queries such as https://domain/registry/users?count=1
+
+            TornadoREST will try to convert the received argument `count` to int.
+
+        '''
+        return Registry.getAllUsers()[:count]
+
+.. note:: For the better look of your requests you can listen default https port (443), balancer like a Nginx can help you with this.
```

Comment:
"for the better look" ?

Reply:
rewrited to `If you need to implement the interface on a standard 443 https port, you will need to use a balancer, such as nginx.`